### DISCUSSION TITLE
Themed icons support

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [zeapo/Android-Password-Store#1911] [Ideas]
Android 12 introduced themed icons as a beta and it looks like this will go gold with Android 13. Would be nice to support it. Several apps do and they look really good with a compatible launcher.

Tasks recently added support and maybe their implemention could be of some use: [Monochrome Icons](https://github.com/tasks/tasks/pull/1876)

**Comment by msfjarvis** [comment]
I'll consider it when it works for me. I have the Signal Beta on my Pixel 4a and on Android 13 Beta 2 I do not see a themed icon, nor did I find any reference within their source repository that a themed icon was ever added.

**Comment by lokesh-krishna** [comment]
It does work for other apps too though. Bromite has it as well. Let's see whether it does for Tasks when they make a release with this commit.
